packages/tocco-ui/src/Upload/helpers.js

```diff
@@ -0,0 +1,5 @@
+export const bytesToSize = bytes => {
+  const units = ['Bytes', 'KB', 'MB', 'GB', 'TB']
+  const i = Math.floor(Math.log(bytes) / Math.log(1024))
+  return (bytes / Math.pow(1024, i)).toFixed(2) * 1 + ' ' + units[i]
```

Comment:
toFixed(2) * 1 --> remove " * 1"

Reply:
since i copied this line i did not know it either but apparently this "* 1" removes unwanted decimal place. e.g without this the function would return "2.00 KB" insted of "2 KB".